pymc/model.py

```diff
@@ -1691,11 +1691,11 @@ def point_logps(self, point=None, round_vals=2):
 
         Parameters
         ----------
-        point: Point
+        point: Point, default 'None'
             Point to be evaluated.  If ``None``, then ``model.initial_point``
             is used.
-        round_vals: int
-            Number of decimals to round log-probabilities
+        round_vals: int, default '2'
```

Comment:
```suggestion
        round_vals: int, default 2
```

The default is the _integer_ 2, not the _string_ '2'. The ticks can be confusing, especially since the type is indicated as int